src/css/loading-blocks.scss

```diff
@@ -0,0 +1,66 @@
+$base-color: #C4C4C4;
+$shine-color: #000;
+$animation-duration: 1s;
+
+@mixin background-gradient {
+  background-color: #fff;
+  background-size: 600px;
+}
+
+@mixin keyframesForLoadingBlocks($numberOfBlocks) {
+  $firstKeyframeValue: floor(floor(100%/$numberOfBlocks)/2);
+  $firstKeyframeValueForCurrentBlock: floor(floor(100/$numberOfBlocks)/2);
+  $firstKeyframeValueForCurrentBlockInPercent: #{$firstKeyframeValueForCurrentBlock} + '%';
+
+  @keyframes blocks_#{$numberOfBlocks}-shine_block1 {
+    0% {
+      background-color: #d1d1d1;
+    }
+
+    #{$firstKeyframeValue} {
+      background-color: #C4C4C4;
+    }
+
+    #{$firstKeyframeValue*2}, 100% {
+      background-color: #d1d1d1;
+    }
+  }
+
+  @if $numberOfBlocks > 1 {
+    @for $i from 2 through $numberOfBlocks {
+      @keyframes blocks_#{$numberOfBlocks}-shine_block#{$i} {
+        0%, #{$firstKeyframeValue+$firstKeyframeValue*($i - 2)} {
+          background-color: #d1d1d1;
+        }
+
+        #{$firstKeyframeValue+$firstKeyframeValue*($i - 1)} {
+          background-color: #C4C4C4;
+        }
+
+        #{$firstKeyframeValue+$firstKeyframeValue*$i}, 100% {
+          background-color: #d1d1d1;
+        }
+      }
+    }
+  }
+}
+
+@include keyframesForLoadingBlocks(1);
+@include keyframesForLoadingBlocks(2);
+@include keyframesForLoadingBlocks(3);
+
+.loading-block-container {
+  display: inline-flex;
+
+  .block {
+    float: left;
+    width: 35px;
+    height: 50px;
+    background-color: #ccc;
```

Comment:
Same here let's use defined colors in the `vars.scss`.

Reply:
And for the others colors in this file.

Reply:
Fixed in https://github.com/keep-network/website/pull/453/commits/3f1544bc8269ddbb364fdc0e166184a1eb99bb49